src/scalper.ts

```diff
@@ -625,6 +641,22 @@ export class Scalper {
       console.log(this.symbol, "Delta Hedge", err, err.stack);
     }
 
+    // Rest single order & do not refresh if the hedge should be completed in a single clip
+    if (spliceFactor == 1){
+      const netHedgePeriod = twapInterval*maxHedges;
+      console.log(this.symbol, "Scan Delta Fills for ~", netHedgePeriod, "seconds");
+      for (let i = 0; i < netHedgePeriod / fillScan; i++) {
+        if (Math.abs(hedgeDeltaTotal * fairValue) < (this.minSpotSize * fairValue)) {
+          this.serumVialClient.removeAnyListeners();
+          console.log(this.symbol, "Delta Hedge Complete: SerumVial");
+          return;
+        }
+        await sleepRandom(fillScan);
```

Comment:
let's try to construct this logic so we dont have to sleep in the delta hedge. if we cant finish in one pass, lets just move on

Reply:
What do you want to do? If you want to just wait til fill, look at what was done in chainlink waiting.

Reply:
Wait until a SerumVial Fill or until the end of the netHedgePeriod (or twapInterval) is over. Right now we get a fill from the websocket but because were already sleeping the thread just does nothing for ahwile. Rather than exiting and moving onto gamma scalping

Reply:
Look at the chainlink waiting and model off of that.